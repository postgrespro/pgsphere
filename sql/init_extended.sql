-- indexed operations.....

-- spoint_data and scircle_data tables have to be created and indexed using

\! perl testsuite/gen_point.pl 1 > results/gen_point_1.sql
\i results/gen_point_1.sql

-- and

\! perl testsuite/gen_circle.pl 1 0.1 > results/gen_circle_1_0.1.sql
\i results/gen_circle_1_0.1.sql

--

\! perl testsuite/gen_poly.pl 1 0.1 4 > results/gen_poly_1_0.1_4.sql
\i results/gen_poly_1_0.1_4.sql
