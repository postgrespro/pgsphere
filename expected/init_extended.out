-- indexed operations.....
-- spoint_data and scircle_data tables have to be created and indexed using
\! perl testsuite/gen_point.pl 1 > results/gen_point_1.sql
\i results/gen_point_1.sql
CREATE TABLE spoint_data (sp spoint);
COPY spoint_data (sp) FROM stdin;
CREATE INDEX sp_idx ON spoint_data USING gist (sp);
-- and
\! perl testsuite/gen_circle.pl 1 0.1 > results/gen_circle_1_0.1.sql
\i results/gen_circle_1_0.1.sql
CREATE TABLE scircle_data (sc scircle);
COPY scircle_data (sc) FROM stdin;
CREATE INDEX sc_idx ON scircle_data USING gist (sc);
--
\! perl testsuite/gen_poly.pl 1 0.1 4 > results/gen_poly_1_0.1_4.sql
\i results/gen_poly_1_0.1_4.sql
CREATE TABLE spoly_data (sp spoly);
COPY spoly_data (sp) FROM stdin;
CREATE INDEX spl_idx ON spoly_data USING gist (sp);
